dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -239,7 +239,16 @@
                 TintOpacity="0.8"
                 FallbackColor="#202020"
                 BackgroundSource="HostBackdrop"/>
```

Comment:
Can you update the non accent acrylic values to latest design POR in this PR or should I do it in a separate PR?

Here is the POR for dark theme:
![image](https://user-images.githubusercontent.com/76921770/106032324-31f52b80-6085-11eb-8635-f95dc6c01d13.png)

Reply:
Sure, I can update the values. Does "Luminosity" translate to the TintLuminosityOpacity property on AcrylicBrushes?

Reply:
Fantastic, thank you!

Yes:
Color % = TintOpacity
Luminocity % = TintLuminosityOpacity
Fallback hex = FallbackColor

Reply:
Thanks, that helps a lot! For the RS2 versions, do we just leave out the luminosity opacity value or should we also adjust the opacity or color?

Reply:
I'm asking Yulia but it looks to me like for rs2 keep the existing TintOpacity (which will be different than the values in the POR and in 19h1) value but update the TintColor and FallbackColor to 19h1 values.

So I think the strategy is the following:

rs2:
TintColor  = POR value
FallbackColor = POR value
TintOpacity = current value in code

19h1:
TintColor  = POR value
FallbackColor = POR value
TintOpacity = POR value
TintLuminosityOpacity = POR value

Reply:
Alright, updated the brushes now.

PS: There's a typo in the inverse brush element (Fallbacak).